src/app/shared/wysiwyg/wysiwyg.component.css

```diff
@@ -10,64 +10,78 @@ div >>> .ProseMirror-menubar-wrapper {
   border: 1px solid #e6e6e6;
 }
 
-div >>> .ProseMirror-content:focus {
+div >>> .ProseMirror:focus {
   color: #000;
   outline: none;
 }
 
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h1,
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h2,
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h3,
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h4,
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h5,
-div >>> .ProseMirror-example-setup-style .ProseMirror-content h6 {
+div.readonly >>> .ProseMirror-menubar-wrapper {
+  border: 1px solid #a3a3a3;
+}
+
+div.readonly >>> .ProseMirror {
+  color: #939393;
+  background-color: #f0f0f0;
+  min-height: 0px;
+}
+
+div.readonly >>> .ProseMirror-menubar {
+  display: none;
```

Comment:
There is a separate view type, EditorView as opposed to the MenuBarEditorView, that doesn't provide a menu, but API is slightly different, so I chose to hide the menu with CSS instead.